04_Fix-Path.md

```diff
@@ -125,44 +90,36 @@ There are two extra path structures that need to be explained:
 * repeated fields
 * arrays
 
-In general: Repeated fields as well arrays are both handled as arrays. They can also call these internal arrays lists.
-Both names (list and array) are reflected in some fix functions (e.g. `add_array` or the `list`-Bind.)
+For both repeated fields and arrays you need to use an **index** to select an element. 
 
-In an data set an element sometimes can have multiple instances. Different data models solve this possibility differently. XML-Records can have all elements multiple times, element repition is possible and in many schemas it is (partly) allowed. E.g. the subject element exists three times:
+For arrays specifically you also need to use an **array marker**.  
```

Comment:
Is the old explanation to complex? In our workshop we made the observation that you have to highlight that different data formats handle repeated fields differently:
- simple repetition (XML)
- arrays (JSON YAML in Practice)
- or both (JSON YAML in principle)
- non at all (tabulated data)